app/javascript/pages/country/data/widgets-config.json

```diff
@@ -72,12 +72,16 @@
       "indicators": ["gadm28", "wdpa", "primary_forests", "plantations", "ifl_2013"],
       "categories": ["summary", "forest-change"],
       "admins": ["country", "region", "subRegion"],
-      "selectors": ["indicators", "thresholds"],
+      "selectors": ["indicators", "thresholds", "extentYears", "units"],
       "type": "gain"
     },
     "settings": {
       "indicator": "gadm28",
-      "threshold": "0"
+      "threshold": "0",
+      "unit": "ha",
+      "extentYear": 2010,
+      "pageSize": 5,
```

Comment:
I guess as we aren't using pages we dont need these. I made an update to the numbered list comp in my branch to handle this better but obviously you dont have this here. Maybe not a concern for now.

Reply:
nice! let me remove it